snippets/csharp/csharp.json

```diff
@@ -56,6 +56,13 @@
     ],
     "description": "Console.WriteLine"
   },
+  "Console.ReadLine": {
+    "prefix": "cr",
+    "body": [
+      "System.Console.ReadLine($0);"
```

Comment:
Why use `System` here? I don't think it's necessary as in new versions of C# it's implicitly used by default. And if not it's one of the first things to do in any program. Might be worth to also drop the `System` for WriteLine.

Reply:
The only reason that it says System.Console instead of Console is because I copy-pasted the WriteLine snippet and changed it to ReadLine. I will edit it on mine when I have the time. Thanks.